.github/workflows/test_suite_ubuntu.yml

```
@@ -62,24 +62,44 @@ jobs:
           . ftorch/bin/activate
           pip install torch torchvision --index-url https://download.pytorch.org/whl/cpu
 
+      # MPI is required by pFUnit
+      - name: Install an MPI distribution
+        run: |
+          sudo apt update
+          sudo apt install mpich
+
+      - name: Install pFUnit
+        run: |
+          export FC=/usr/bin/gfortran
+          export MPIF90=/usr/bin/mpif90
+          # TODO: Avoid version pinning (needed because version appears in install path)
+          git clone -b v4.10.0 https://github.com/Goddard-Fortran-Ecosystem/pFUnit.git
+          mkdir pFUnit/build
+          cd pFUnit/build
+          cmake -DMPI=YES ..
```

```suggestion
          cmake ..
```

I think this would be OK, but can't test locally on my machine at present.
Looking at [the docs](https://github.com/Goddard-Fortran-Ecosystem/pFUnit/tree/v4.10.0) it seems there is a `SKIP_MPI` option which we would not want if building with MPI support.